web/src/components/molecules/Common/projectMenu/index.tsx

```diff
@@ -41,10 +41,16 @@ const ProjectMenu: React.FC<Props> = ({
   ];
 
   const onClick = (e: any) => {
-    if (e.key === "home") {
-      navigate(`/dashboard/${workspaceId}`);
-    } else if (e.key === "schema") {
-      navigate(`/workspaces/${workspaceId}/${projectId}/schema`);
+    switch (e.key) {
```

Comment:
Can you type this onClick?

Reply:
I think that there is no problem with using switch statement inside onClick function.